tools/mo/requirements.txt

```diff
@@ -9,4 +9,5 @@ defusedxml>=0.7.1
 urllib3>=1.26.4
 requests>=2.25.1
 fastjsonschema~=2.15.1
+openvino-telemetry>=2022.1.0
```

Comment:
why not openvino-telemetry~=2022.1.0? I am not sure that OpenVINO will be compatible with future releases of the telemetry.

Reply:
During the discussion on Integration forum there was a requirement that telemetry should have backward compatible versions. Otherwise there might be conflicts when different packages use different versions of telemetry. So it should be possible to specify the highest telemetry version for all packages.

Reply:
So telemetry package should guarantee backward compatibility.